cl/opinion_page/urls.py

```diff
@@ -27,6 +28,22 @@
         view_docket,
         name="view_docket"
     ),
+    url(
+        r'''(?x)
```

Comment:
```
$ pydoc re
        (?iLmsux) Set the I, L, M, S, U, or X flag for the RE (see below).
        X  VERBOSE     Ignore whitespace and comments for nicer looking RE's.
```

The `x` flag allows splitting the RE up over multiple lines which avoids having `flake8` complain about a line greater than 80cols.